faunadb-scala/src/test/scala/faunadb/SerializationSpec.scala

```diff
@@ -362,7 +362,7 @@ class SerializationSpec extends FlatSpec with Matchers {
     TimeV(Instant.ofEpochMilli(0)).toString shouldBe "1970-01-01T00:00:00Z"
     DateV(LocalDate.ofEpochDay(0)).toString shouldBe "1970-01-01"
     BytesV("DEADBEEF").toString shouldBe "[0x0c 0x40 0x03 0x04 0x41 0x05]"
-    QueryV(ObjectV("lambda" -> "_", "expr" -> true)).toString shouldBe "{lambda: \"_\", expr: true}"
+    QueryV(ObjectV("lambda" -> "_", "expr" -> true, "api_version" -> "3")).toString shouldBe """{lambda: "_", expr: true, api_version: "3"}"""
```

Comment:
that test in specific is not testing serialization, it's testing `toString` method

Reply:
Thanks for the hint, Marrony! Since they are not covering the serialization, it might make sense to move those tests to another suite  to avoid future confusions.